routes/route.js

```diff
@@ -6,8 +6,25 @@ router.get('/check', (req, res) => {
     res.send("Congratulations! Your app works! :)");
 })
 
-router.post('add', (req, res) => {
-    // Add logic here
+router.post('/add', (req, res) => {
+    //ADD LOGIC
+    
+
+    //MULTIPLICATION LOGIC
+    router.get('/multiply', (req, res) => {
+        if (req.body.x && req.body.y) {                                         //checking for existance both the numbers
+            if ((!isNaN(req.body.x)) && (!isNaN(req.body.y))) {                 //checking for the values for both the numbers
+                const multiplication = parseInt(req.body.x) * parseInt(req.body.y);
+                res.status(200).send(`${req.body.x} * ${req.body.y} = ${multiplication}`);
+            } else {
+                res.status(400).send('The arguments are not numbers, Please enter valid arguments');
+            }
+        } else {
+            res.status(400).send('Both arguments were not provided, Please provide valid arguments');
+        }
+    })
```

Comment:
You are trying to write a route inside another route (`/add`) which is fundamentally wrong. Can you please fix this. Thanks